proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -206,6 +206,674 @@ function bufferNodeIs( test )
+  test.case = 'check empty U64x';
+  var src = new U64x( 2 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'check U8x';
+  var src = new U8x( [ 2, 3 ] );
+  var got = _.bufferBytesIs( src );
+  var expected = true;
+
+  test.identical( got, expected );
+  test.case = 'check U16x';
+  var src = new U16x( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check U32x';
+  var src = new U32x( [ 2, 3 ] );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check U64x';
+  var src = new U64x( _.bigIntsFrom( [ 2, 3 ] ) );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'check empty I8x';
+  var src = new I8x( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check empty I16x';
+  var src = new I16x( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check empty I32x';
+  var src = new I32x( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check empty I64x';
+  var src = new I64x( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'check I8x ';
+  var src = new I8x( [ 2, 3 ] );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check I16x';
+  var src = new I16x( [ 2, 3 ] );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check I32x';
+  var src = new I32x( [ 2, 3 ] );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check I64x';
+  var src = new I64x( _.bigIntsFrom( [ 2, 3 ] ) );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'check empty F32x';
+  var src = new F32x( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check F32x';
+  var src = new F32x( [ 2, 3 ] );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'check empty F64x';
+  var src = new F32x( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check F64x';
+  var src = new F64x( [ 2, 3 ] );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /* BufferNode */
+
+  if( Config.interpreter === 'njs' )
+  {
+    test.case = 'check empty BufferNode';
+    var src = new BufferNode.alloc(5);
```

Comment:
Please, see code style. 
The before and after argument should be spaces.

For quick searching and replacing use functionality of your editor